.rubocop_todo.yml

```diff
@@ -53,14 +53,14 @@ Layout/BlockAlignment:
 # Cop supports --auto-correct.
 Layout/ClosingParenthesisIndentation:
   Exclude:
```

Comment:
All of the changes in this file are because we needed to rename the previous file to the new file inside of /lib.